client/framework/esx/compatibility.lua

```diff
@@ -41,6 +41,7 @@ RegisterNetEvent("skinchanger:loadSkin", function(skin, cb)
 	end
 end)
 
+
```

Comment:
Can you please remove the extra blank line? There shouldn't be more than 1 blank line between functions and events

Reply:
alright